### PR TITLE
CVE-2003-0253

### PR DESCRIPTION
**Pull request by akn4743**

### CODE REVIEW

File: cves/CVE-2003-0253.yml [hunk 1]
```diff
@@ -3,22 +3,22 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 703
```
Comment (JO4IVIV4) [inline cves/CVE-2003-0253.yml]: It would be useful to include the name of the cwe

File: cves/CVE-2003-0253.yml [hunk 2]
```diff
@@ -208,4 +219,11 @@ mistakes:
...
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    A lot of factors came to allow this vulnerability to occur. The original author
+    wasnt confident in the comments that the code would work all of the time. Edge 
+    cases in the code weren't properly met. Its possible that there was confusion 
+    in the requirements and the task wasnt descriptive enough to who was working on 
+    it. Unit tests were not made to ensure it the code worked as intended. 
+    The mitigations for this fix on the CWE was to run fault injection tests.
```
Comment (JO4IVIV4) [inline cves/CVE-2003-0253.yml]: You should mention whether the mitigations on the CWE were applied

Comment (akn4743) [reply cves/CVE-2003-0253.yml]: Made changes to the end of the paragraph to include whether the mitigation was applied.

File: cves/CVE-2003-0253.yml [hunk 3]
```diff
@@ -32,7 +32,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  MPM's are multi-processing modules which are used to bind to network ports on a 
+  machine, accept requests, and handle requests. This vulnerability occurs when
+  the MPM binds to a specific uncommonly used network port. When the C accept() method
```
Comment (17mgeffert) [inline cves/CVE-2003-0253.yml]: it is unclear what the accept()  method does, you could/should explain it in detail

File: cves/CVE-2003-0253.yml [hunk 4]
```diff
@@ -103,11 +107,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
+  answer: | 
+    The problem was reported by Saheed Akhtar. No mention on how it was found.
```
Comment (17mgeffert) [inline cves/CVE-2003-0253.yml]: Is there any information on where it was found? like if it was posted in some thread somewhere at anypoint? or does that fall under "no mention on how it was found"

**Comment by PaceyLong** [comment]
I think the written is fine

**Pull request merged**